app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -199,13 +199,15 @@ def contextual_path(presenter, parent_presenter)
       ::Hyrax::ContextualPath.new(presenter, parent_presenter).show
     end
 
-    def curation_concern_from_search_results
```

Comment:
can we deprecate this, rather than dropping it?

i've been considering `private` methods on these mixins public API, since they are only "private" to the scope of controllers which users' applications own directly.